package.json

```
@@ -62,5 +64,10 @@
       "type": "Apache-2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0"
     }
-  ]
+  ],
+  "storybook-deployer": {
+    "gitUsername": "Vitamin BOT",
+    "gitEmail": "designsystem@decathlon.net",
+    "commitMessage": "chore: deploy Storybook"
```

```suggestion
    "commitMessage": "chore: deploy storybook"
```
Because we use the default commitlint configuration which uses camelCase by default so "Storybook" must be in lowercase. See https://github.com/conventional-changelog/commitlint/blob/master/docs/reference-rules.md#header-case

---

To my mind, ci is more when you do actions for the CI and not for the commits done in the CI but it ca' be discussed

---

Actually that's not a big deal because these commits will only be present on the gh-pages branch

---

Oh yes okay! So, no problem :)